ReleaseNotes.md

```diff
@@ -0,0 +1,14 @@
+# Release Notes for IIT Floor Finder
+
+## Floor Finder 0.0.0
```

Comment:
Maybe we could use additional labels for pre-release. For examples: 1.0.0-alpha

Reply:
Done!